qvm/sys-gui-gpu-attach-gpu.sls

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# qvm.sys-gui-gpu-vm-attach-gpu
+# ===========
+##
+
+sys-gui-gpu-down:
+  qvm.shutdown:
+    - name: sys-gui-gpu
+
+sys-gui-gpu-attach-gpu:
+  cmd.run:    
+    - name: qvm-pci attach -o persistent=True -o permissive=True sys-gui-gpu dom0:00_02.0
```

Comment:
There is `qvm.devices` state for this, see docstring for example: https://github.com/QubesOS/qubes-mgmt-salt-dom0-qvm/blob/master/_modules/ext_module_qvm.py#L978-L990